.github/workflows/test-all.yml

```diff
@@ -0,0 +1,147 @@
+name: test-all
+
+on:
+  schedule:
+    - cron: '0 1 * * *'   # 1 AM UTC daily
+  repository_dispatch:
+    types: [unikraft_pr]  # Triggered by external PRs in the unikraft repository
```

Comment:
Let's rename it to action specific `opened_pr`, `unikraft_pr` is a generic name.